pacu/modules/ebs__enum_snapshots_unauth/main.py

```diff
@@ -0,0 +1,158 @@
+     keywords/account IDs from wordlists. The results will be saved in the session's directory and printed to
+     the console in the summary if there are 25 or fewer results.'''),
+    'services': ['EC2'],
+    'prerequisite_modules': [],
+    'arguments_to_autocomplete': ['--keyword', '--account-id', '--keyword-wordlist', '--account-id-wordlist'],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info['description'])
+
+
+parser.add_argument(
+    '--keyword',
+    help='Keyword to search for in EBS snapshot descriptions.'
+)
+parser.add_argument(
+    '--account-id',
+    help='AWS account ID to search for associated EBS snapshots.'
+)
+parser.add_argument(
+    '--keyword-wordlist',
+    help='File path to a wordlist of keywords to search for in EBS snapshot descriptions.'
+)
+parser.add_argument(
+    '--account-id-wordlist',
+    help='File path to a wordlist of AWS account IDs to search for associated EBS snapshots.'
+)
+
+
+def load_wordlist(file_path):
+    try:
+        with open(file_path, 'r') as file:
+            return file.read().splitlines()
+    except Exception as e:
+        return str(e)
+
+
+def search_snapshots(ec2_client, filters):
+    try:
+        return ec2_client.describe_snapshots(Filters=filters).get('Snapshots', [])
+    except ClientError as error:
+        print('    FAILURE:')
+        print('    ' + error.response['Error']['Code'])
+        return []
+
+
+def main(args, pacu_main: 'Main'):
+    session = pacu_main.get_active_session()
+    args = parser.parse_args(args)
+    print = pacu_main.print
+    get_regions = pacu_main.get_regions
+
+    ec2_data = deepcopy(session.EC2) if hasattr(session, 'EC2') else {}
+    if 'Volumes' not in ec2_data.keys():
+        ec2_data['Volumes'] = []
+    if 'Snapshots' not in ec2_data.keys():
+        ec2_data['Snapshots'] = []
+
+    existing_snapshots_ids = {snapshot['SnapshotId'] for snapshot in ec2_data['Snapshots']}
+    snapshots_found = []  # Store all the snapshots found during the current execution
+
+    keyword_list = [args.keyword] if args.keyword else load_wordlist(
+        args.keyword_wordlist) if args.keyword_wordlist else []
+    account_id_list = [args.account_id] if args.account_id else load_wordlist(
+        args.account_id_wordlist) if args.account_id_wordlist else []
+
+    if not keyword_list and not account_id_list:
+        print("Please specify a keyword, account ID, keyword wordlist, or account ID wordlist.")
+        return
+
+    regions = get_regions('ec2')
+    if not regions:
+        print("No regions found. Exiting...")
+        return
+
+    session_name = session.name
+    directory = str(downloads_dir() / session_name / 'ebs')
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+    for region in regions:
+        print('[ebs__enum_snapshots_unauth] Starting region {}...'.format(region))
```

Comment:
remove the preceding "[ebs__enum_snapshots_unauth] " this is already handled by Pacu print()